en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -1,193 +1,212 @@
 
-This guide explains how to configure and use email domain-based organization discovery in {{ product_name }}.
+This guide explains how you may use {{product_name}} to configure email domain-based organization discovery for your B2B applications.
 
 ## Scenario
 
-Consider **Guardio Insurance** is a B2B SaaS provider and `Best Car Mart` and `Glory Car Mart` are two of its customer/partner organizations.
-Guardio Insurance wants to resolve the organization of a user based on the email domain of the user's email address and route them to the respective authentication options.
+**Guardio Insurance** is an insurance provider. *Best Car Mart* and *Glory Car Mart* are two of its customer organizations. When a user enters an email address to log into the Guardio Insurance platform, it is required to resolve a user's organization based on the email domain and redirect them to the respective organization's login screen.
+
 
 ![Email domain based organization discovery scenario]({{base_path}}/assets/img/guides/organization/manage-organizations/email-domain-based-organization-discovery.png){: width="700" style="display: block; margin: 0;"}
 
-User Alice with email `alice@bcmart.com` should be routed to the authentication options provided by the `Best Car Mart` organization,
-and user Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be routed to the authentication options provided by the `Glory Car Mart` organization.
+In this example scenario,
+
+- User Alice with email `alice@bcmart.com` should be routed to the login screen of `Best Car Mart` organization.
+- User Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be routed to the login screen of `Glory Car Mart` organization.
 
 ## Enable email domain based organization discovery
 
-!!! note
-    - This feature can only be used when email address is configured as the username.
-    - Enabling email domain based organization discovery feature, and adding email domain mapping against the organizations is permitted only for the organization (root).
+!!! note "Important"
+    Enabling this feature and mapping email domains to organizations is only permitted in the root organization. 
 
-### Using the Console
+=== "Using the Console"
 
-1. Login to the organization (root) from the {{ product_name }} Console.
-2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Organization Discovery** under **Organization Settings**.
-3. Turn on the toggle to enable email domain based organization discovery.
+    !!! note "Before you begin"
 
-    !!! note
-        When this is enabled, following restrctions will apply to child organizations during federated authentication and user onboarding.
+        Ensure you have configured email address as a username option. Learn more about [multi-attribute login]({{base_path}}/guides/authentication/multi-attribute-login/).
```

Comment:
Let's move this as mentioned above. Also, email should be configured as in this doc: https://is.docs.wso2.com/en/latest/guides/users/attributes/enable-email-as-username/
not multi attribute login

Reply:
fixed with 3033eb4d1d38180c4ca1d0ad42b202f75b46363f